### PR TITLE
soap-seed-fault

### PR DESCRIPTION
**Pull request by cpa-aida**
#### Features
- services-soap: seed creation for soap faults

### CODE REVIEW

File: packages/services-soap/src/mode/seed.ts [hunk 1]
```diff
@@ -48,14 +54,42 @@ export class SoapClientSupplierSeed extends SoapClientSupplierBase {
...
   }
 
+  private handleError(seed: FxSt<any>, error: any) {
+    const isSoapFault =
+      error &&
+      error.cause &&
+      error.cause.root &&
+      error.cause.root.Envelope &&
+      error.cause.root.Envelope.Body &&
+      error.cause.root.Envelope.Body.Fault
```
Comment (Aduril) [inline packages/services-soap/src/mode/seed.ts]: 😭

File: packages/services-soap/src/mode/seed.ts [hunk 2]
```diff
@@ -48,14 +54,42 @@ export class SoapClientSupplierSeed extends SoapClientSupplierBase {
...
+    if (isSoapFault) {
+      const code = error.cause.root.Envelope.Body.Fault.faultcode
+      const description = error.cause.root.Envelope.Body.Fault.faultstring
+      const details = error.cause.root.Envelope.Body.Fault.detail
```
Comment (Aduril) [inline packages/services-soap/src/mode/seed.ts]: You alreardy have the Fault object in isSoapFault. Use it ^^

File: packages/services-soap/src/mode/seed.ts (hunk 1 above)
Comment (cpa-aida) [reply packages/services-soap/src/mode/seed.ts]: i'll think about introducing the 'tryGet' function from ... other projects next time ;)

File: packages/services-soap/src/mode/seed.ts (hunk 2 above)
Comment (cpa-aida) [reply packages/services-soap/src/mode/seed.ts]: true

**Pull request merged**